### DISCUSSION TITLE
[taskbar-clock-customization] Using only one line in the taskbar clock

### DISCUSSION THREAD

**Discussion by alifputras** [ramensoftware/windhawk-mods#535] [Q&A]
How do you remove the bottom line if i only wanted to use one line to make it look nicer? Or maybe just center the top line?

I'm trying to remove the bottom line and it looks bad because of an empty space under the top line.
![Screenshot 2024-02-20 201825](https://github.com/ramensoftware/windhawk-mods/assets/75693282/076cf458-4638-42f1-95c9-9a072a5fadce)

**Comment by m417z** [comment]
You can hide the top line, then customize the bottom line as desired.

![image](https://github.com/ramensoftware/windhawk-mods/assets/4129781/49259dc7-f063-44e8-a496-acdc162acb1d)
